bundles/org.openhab.binding.mqtt.ruuvigateway/src/main/resources/OH-INF/i18n/mqtt.properties

```
@@ -0,0 +1,42 @@
+# thing types
+
+thing-type.mqtt.ruuvitag_beacon.label = RuuviTag SmartBeacon
+thing-type.mqtt.ruuvitag_beacon.description = A RuuviTag SmartBeacon
+
+# thing types config
+
+thing-type.config.mqtt.ruuvitag_beacon.topic.label = MQTT topic
+thing-type.config.mqtt.ruuvitag_beacon.topic.description = MQTT topic containing the payload
+
+# channel types
+
+channel-type.mqtt.ruuvitag_accelerationx.label = Acceleration X
+channel-type.mqtt.ruuvitag_accelerationy.label = Acceleration Y
+channel-type.mqtt.ruuvitag_accelerationz.label = Acceleration Z
+channel-type.mqtt.ruuvitag_batteryVoltage.label = Battery Voltage
+channel-type.mqtt.ruuvitag_dataFormat.label = Data Format Version
+channel-type.mqtt.ruuvitag_gwmac.label = Gateway MAC Address
+channel-type.mqtt.ruuvitag_gwmac.description = MAC-address of Ruuvi Gateway
+channel-type.mqtt.ruuvitag_gwts.label = Relay Timestamp
+channel-type.mqtt.ruuvitag_gwts.description = Timestamp when the message from Bluetooth sensor was relayed by Gateway (gwts)
+channel-type.mqtt.ruuvitag_gwts.state.pattern = %1$tY-%1$tm-%1$td %1$tH:%1$tM:%1$tS
+channel-type.mqtt.ruuvitag_humidity.label = Humidity
+channel-type.mqtt.ruuvitag_measurementSequenceNumber.label = Measurement Sequence Number
+channel-type.mqtt.ruuvitag_movementCounter.label = Movement Counter
+channel-type.mqtt.ruuvitag_pressure.label = Pressure
+channel-type.mqtt.ruuvitag_rssi.label = RSSI
+channel-type.mqtt.ruuvitag_rssi.description = Received signal strength indicator
+channel-type.mqtt.ruuvitag_temperature.label = Temperature
+channel-type.mqtt.ruuvitag_ts.label = Timestamp
+channel-type.mqtt.ruuvitag_ts.description = Timestamp when the message from Bluetooth sensor was received by Gateway (ts)
+channel-type.mqtt.ruuvitag_ts.state.pattern = %1$tY-%1$tm-%1$td %1$tH:%1$tM:%1$tS
+channel-type.mqtt.ruuvitag_txPower.label = TX Power
+
+# Thing status messages
+
+online.waiting_initial_data = Waiting for initial data
+offline.configuration-error.missing_topic = Missing topic configuration, cannot subscribe to relevant MQTT topic
```

Very minor, but perhaps it would be more consistent to use only dashes as opposed to mixing dashes and underscores:
```suggestion
offline.configuration-error.missing-topic = Missing topic configuration, cannot subscribe to relevant MQTT topic
```